apps/opik-backend/src/main/java/com/comet/opik/infrastructure/EncryptionService.java

```diff
@@ -0,0 +1,61 @@
+package com.comet.opik.infrastructure;
+
+import com.google.inject.ImplementedBy;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.NonNull;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Base64;
+
+@ImplementedBy(EncryptionServiceImpl.class)
+public interface EncryptionService {
+    String encrypt(String data);
+    String decrypt(String encryptedData);
+}
+
+@Singleton
+class EncryptionServiceImpl implements EncryptionService {
+
+    private static final String ALGO = "AES";
+    private final Base64.Encoder mimeEncoder = Base64.getMimeEncoder();
+    private final Base64.Decoder mimeDecoder = Base64.getMimeDecoder();
```

Comment:
Minor: according to the Javadoc, these are thread safe, so they can be static and final. Anyway, in practice they're as this is singleton class.

Reply:
done